fuzzing/fuzzer_test.go

```diff
@@ -590,8 +590,10 @@ func TestCorpusReplayability(t *testing.T) {
 			assertCorpusCallSequencesCollected(f, true)
 			newCoverage := f.fuzzer.corpus.CoverageMaps()
 
-			// Check to see if original and new coverage are the same
-			assert.True(t, originalCoverage.Equals(newCoverage))
+			// Check to see if original and new coverage are the same.
+			// TODO: Fix this, as coverage now tracks reverted too, and if this test solved too quickly on first run,
```

Comment:
was this meant to leave as a TODO ?

Reply:
This was removed in https://github.com/crytic/medusa/pull/111/commits/9e77c357bb5c06abd8992863c3495922b08fc674